File: app/src/main/java/android/itacademy/by/dz6/recycle/ImageLoaderUtil.java
package android.itacademy.by.dz6.recycle;

import android.itacademy.by.menu.R;
import android.widget.ImageView;

import com.squareup.picasso.Picasso;

public class ImageLoaderUtil {
    public static void loadImage(ImageView imageView,String url){
//        Picasso.get().load(url).into(imageView);

        Picasso.get()
                .load(url)
                .placeholder(R.drawable.progress)
                .into(imageView);

    }
}

File: app/src/main/java/android/itacademy/by/dz6/recycle/ImageLoaderUtil.kt
package android.itacademy.by.dz6.recycle

import android.widget.ImageView

import com.squareup.picasso.Picasso

object ImageLoaderUtil {
    fun loadImage(imageView: ImageView, url: String) {
        Picasso.get().load(url).into(imageView)
    }
}